src/shutdown/umount.h

```diff
@@ -20,6 +20,7 @@ typedef struct MountPoint {
         unsigned long remount_flags;
         bool try_remount_ro:1;
         bool umount_lazily:1;
+        bool leaf:1;
```

Comment:
Oh, that's something to fix independently. Please make a separate commit to drop the bitfield markers. They make the code text more complicated but don't actually save any space:
```
struct MountPoint {
        char *                     path;                 /*     0     8 */
        char *                     remount_options;      /*     8     8 */
        long unsigned int          remount_flags;        /*    16     8 */
        _Bool                      try_remount_ro:1;     /*    24: 0  1 */
        _Bool                      umount_lazily:1;      /*    24: 1  1 */

        /* XXX 6 bits hole, try to pack */
        /* XXX 7 bytes hole, try to pack */

        dev_t                      devnum;               /*    32     8 */
```

Reply:
Done.